commit 4f660fcf1984e21a5adb8ab350ef579b91a14eea (refs/changes/78/2778/3)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2017-10-13T14:34:27+00:00 (2 years ago)
    
    OS-6396 LTP new testcase ioctl07 fails
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/usr/src/lib/brand/lx/testing/ltp_skiplist b/usr/src/lib/brand/lx/testing/ltp_skiplist
index 7340a37b58..0f670d1866 100644
--- a/usr/src/lib/brand/lx/testing/ltp_skiplist
+++ b/usr/src/lib/brand/lx/testing/ltp_skiplist
@@ -171,6 +171,7 @@ ioctl03
 ioctl04
 ioctl05
 ioctl06
+ioctl07			# OS-6397
 keyctl01
 keyctl02
 keyctl03
